Makefile

```diff
@@ -290,7 +290,7 @@ azure:
 	$(MAKE) apply WHAT=projects/azure/cluster-wide.yaml
 	$(MAKE) apply WHAT=projects/azure/rbac.yaml
 	# ci namespace objects
-	oc create secret generic cluster-secrets-azure-file --from-file=cluster/test-deploy/azure/secret --from-file=cluster/test-deploy/azure/ssh-privatekey -o yaml --dry-run | oc apply -n ci -f -
+	oc create secret generic cluster-secrets-azure-file --from-file=cluster/test-deploy/azure/secret --from-file=cluster/test-deploy/azure/ssh-privatekey --from-file=cluster/test-deploy/azure/certs.yaml -o yaml --dry-run | oc apply -n ci -f -
```

Comment:
Eventually we can stop creating directly in the ci namespace, cf https://github.com/openshift/release/issues/1634